elasticdl/requirements.txt

```diff
@@ -3,4 +3,4 @@ kubernetes
 docker
 pyrecordio>=0.0.6
 odps
-redis-py-cluster
\ No newline at end of file
+redis-py-cluster
```

Comment:
Why redis-py-cluster is still needed?

Reply:
oh, I think here I just add a new blank line there. `redis-py-cluster` is there before.. Not sure whether we have removed all redis code.